taskt/Core/Automation/Commands/Data Commands/DateCalculationCommand.cs

```diff
@@ -186,7 +176,7 @@ public override string GetDisplayValue()
                 }
 
                 //return value
-                return base.GetDisplayValue() + " [" + operand + " " + v_Increment + " " + interval + operandLanguage + v_InputValue + ", Apply Result to Variable '" + v_applyToVariableName + "']";
+                return base.GetDisplayValue() + $" [{operand} {v_Increment} {interval} {operandLanguage} {v_InputDate} - Apply Result to Variable '{v_OutputUserVariableName}']";
```

Comment:
Please remove extra spaces in interval string. Also "Apply Result to Variable" should be "Store Date in" as per the guideline template

Reply:
Done.
Actually, extra spaces were in **operandLanguage** string